packages/manager/apps/web/client/app/hosting/freedom/FREEDOM.html

```diff
@@ -116,6 +116,7 @@ <h2 data-translate="hosting_tab_FREEDOM"></h2>
                             <oui-action-menu-item
                                 data-on-click="ctrlHostingFreedomTab.setSelectedProduct(freedom.domain)"
                                 data-href="{{'#/domain/'+freedom.domain}}"
+                                top="true"
```

Comment:
Not required. We need to use `_top` only for external URL or url builded with URLBuilder
We need to check more in details.